sse2neon.h

```diff
@@ -2023,6 +2023,15 @@ FORCE_INLINE int _mm_movemask_ps(__m128 a)
 #endif
 }
 
+// Check if input 'a' is all 1's, and return 0 if the 'a' is all 1's, otherwise
+// return 1.
```

Comment:
You should provide the description of [_mm_test_all_ones](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_test_all_ones).

Reply:
I will replace the current one with the description in Intel Intrinsics page